database/migrations/2021_04_06_045402_create_roles_table.php

```diff
@@ -18,6 +19,20 @@ public function up()
             $table->string('title');
             $table->timestamps();
         });
+        DB::table('roles')->insert(
+            [
+                [
+                    'title' => 'student',
+                    'created_at' =>  \Carbon\Carbon::now(),
+                    'updated_at' => \Carbon\Carbon::now(),
+                ],
+                [
+                    'title' => 'admin',
+                    'created_at' =>  \Carbon\Carbon::now(),
+                    'updated_at' => \Carbon\Carbon::now(),
+                ]
+            ]
+        );
```

Comment:
Please use a database seeder to achieve this.

https://laravel.com/docs/8.x/seeding